HospitalWebApp/Resources/verification-mail.html

```diff
@@ -0,0 +1,267 @@
+                                      </tr>
+                                    </tbody>
+                                  </table>
+                                </td>
+                              </tr>
+                            </tbody>
+                          </table><span style="font-weight: 600"> <i18n-string data-key="account.verifyUserEmailAddress.ifNewSignupText" data-locale-at-render="en-us">Thanks for signing up to HealthCare Web Application. We’re happy to have you!</i18n-string> </span>
+                          <p style="margin: 0">
+                            <i18n-string data-key="account.verifyUserEmailAddress.bodyText" data-locale-at-render="en-us">Please take a second to make sure we have your correct email address.</i18n-string>
+                          </p>
+                          <table border="0" cellpadding="0" cellspacing="0" width="100%" style="font-size: 0; height: 50px; width: 100%; min-width: 100%; line-height: 0;">
+                            <tbody>
+                              <tr>
+                                <td height="50"><span style="-webkit-text-size-adjust:100%; -ms-text-size-adjust:100%; color: transparent; background: none; user-select: none; -moz-user-select: none; -ms-user-select:none; -webkit-user-select:none; text-overflow: ellipsis; opacity: 0; width:100%; min-width: 100%; height:1; overlfow:hidden; margin: -1px 0 0 0; padding:0; font-size: 0;"> &nbsp;</span></td>
+                              </tr>
+                            </tbody>
+                          </table>
+                          <table width="100%" border="0" cellspacing="0" cellpadding="0">
+                            <tbody>
+                              <tr>
+                                <td align="center">
+                                  <table border="0" cellspacing="0" cellpadding="0">
+                                    <tbody>
+                                      <tr>
+                                        <td align="center" style="border-radius: 3px;" bgcolor="#425b76" width="auto"><a href="#" id="activationPath"
+                                            target="_blank" style="border: 1px solid #425b76; border-radius: 3px; color: #FFFFFF; display: inline-block; font-size: 14px; font-weight: 500; line-height: 1; padding: 12px 20px; text-decoration: none; width: auto; min-width: 170px; white-space: nowrap; ">Confirm your email address</a></td>
+                                      </tr>
+                                    </tbody>
+                                  </table>
+                                </td>
+                              </tr>
+                            </tbody>
+                          </table>
+                          <div itemscope itemtype="http://schema.org/EmailMessage">
+                            <div itemprop="potentialAction" itemscope itemtype="http://schema.org/ViewAction">
```

Comment:
Could you define style for all tags in separate css file ?

Reply:
Yes, but this way of defining styles is much more suitable for this functionalty.
I if wanted to define styles in a separate file I would have to merge html and css file at some point.